tests/integration/hpe_3par_api_multi_array_test.py

```diff
@@ -0,0 +1,851 @@
+        # Verifying in 3par
+        self.hpe_verify_volume_unmount(volume_name)
+        self.hpe_inspect_container_volume_unmount(volume_name, container_name)
+        self.client.remove_container(container_id)
+
+        self.hpe_delete_snapshot(volume_name, snapshot_name)
+        self.hpe_verify_snapshot_deleted(volume_name, snapshot_name)
+
+        self.hpe_delete_volume(volume)
+        self.hpe_verify_volume_deleted(volume_name)
+
+
+    def test_multi_array_support(self):
+        volume_name = helpers.random_name()
+        self.tmp_volumes.append(volume_name)
+
+        snapshot_name = helpers.random_name()
+        self.tmp_volumes.append(snapshot_name)
+
+        clone_name = helpers.random_name()
+        self.tmp_volumes.append(clone_name)
+
+        container_name= helpers.random_name()
+        self.tmp_containers.append(container_name)
+
+        volume = self.hpe_create_volume(volume_name, driver=HPE3PAR, 
+                                        size=THIN_SIZE, provisioning='full',
+                                        backend="backend2"
+                                       )
+        self.hpe_verify_volume_created(volume_name, provisioning='full', 
+                                       size=THIN_SIZE, backend="backend2")
+        self.hpe_inspect_volume(volume, size=int(THIN_SIZE), 
+                                provisioning='full', backend="backend2")
+
+        snapshot = self.hpe_create_snapshot(snapshot_name, driver=HPE3PAR,
+                                            virtualCopyOf=volume_name)
+        self.hpe_inspect_snapshot(snapshot, snapshot_name=snapshot_name,
+                               virtualCopyOf=volume_name, size=int(THIN_SIZE),
+                               backend="backend2")
+        self.hpe_verify_snapshot_created(volume_name, snapshot_name,
+                                         backend="backend2")
+
+        clone = self.hpe_create_volume(clone_name, driver=HPE3PAR,
+                                      cloneOf=volume_name)
+        self.hpe_inspect_volume(clone, size=int(THIN_SIZE),
+                                provisioning='full')
+        self.hpe_verify_volume_created(clone_name, size=THIN_SIZE,
+                                       provisioning='full', clone=True, 
+                                       backend="backend2")
+
+        host_conf = self.hpe_create_host_config(volume_driver=HPE3PAR,
+                                               binds= volume_name + ':/data1')
+        container_info = self.hpe_mount_volume(BUSYBOX, command='sh', detach=True,
+                              tty=True, stdin_open=True,
+                              name=container_name, host_config=host_conf
+                              )
+
+        container_id = container_info['Id']
+        self.hpe_inspect_container_volume_mount(volume_name, container_name)
+        # Verifying in 3par
+        self.hpe_verify_volume_mount(volume_name, backend="backend2")
```

Comment:
This verification is failing because vlun_cnt = 1

Reply:
You are testing with FC setup but automation framework designed for ISCSI protocol. Please test with ISCSI setup.